web_responsive/static/src/js/web_responsive.js

```diff
@@ -276,6 +285,20 @@ odoo.define('web_responsive', function (require) {
         },
     });
 
+    BasicController.include({
+        
+        /**
+         * @override
+         */
+        canBeDiscarded: function (recordID) {
+            if (this.model.isDirty(recordID || this.handle)) {
```

Comment:
Also here. Code is not obvious.

Reply:
https://github.com/OCA/web/commit/ce0391a0a0a21de4cc3caffa6829a99437e3db0e